src/main/webapp/community.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Community Page</title>
+    <link rel="stylesheet" href="/css/main.css">
+
+    <script>
+
+      /** Fetches users and adds them to the page. */
+      function fetchUserList(){
+        const url = '/user-list';
+        fetch(url).then((response) => {
+          return response.json();
+        }).then((users) => {
+          const list = document.getElementById('list');
+          list.innerHTML = '';
+
+          users.forEach((user) => {
+           const userListItem = buildUserListItem(user);
+           list.appendChild(userListItem);
+          });
+        });
+      }
+
+      /**
+       * Builds a list element that contains a link to a user page, e.g.
+       * <li><a href="/user-page.html?user=test@example.com">test@example.com</a></li>
+       */
+      function buildUserListItem(user){
+        const userLink = document.createElement('a');
+        userLink.setAttribute('href', '/user-page.html?user=' + user);
+        userLink.appendChild(document.createTextNode(user));
+        const userListItem = document.createElement('li');
+        userListItem.appendChild(userLink);
+        return userListItem;
+      }
+
+      /** Fetches data and populates the UI of the page. */
+      function buildUI(){
```

Comment:
_Just fyi hehe_ 
It is normally better to add scripts to the bottom of the page, so the body loads as soon as it can and then it gets populated with data by the script, but since it is all done onload, it will probably be almost if not the same.

Reply:
This was just a comment, so no need for changes.